src/extensions/default/AutoUpdate/main.js

```diff
@@ -158,8 +181,22 @@ define(function (require, exports, module) {
                     title: Strings.UPDATE_SUCCESSFUL,
                     description: ""
                 });
+                HealthLogger.sendAnalyticsData(AUTOUPDATE_INSTALLATION_SUCCESS, "autoUpdate", "install", "complete", "");
             } else {
                 // We get here if the update started but failed
+                var searchParams = {
+                    "updateDir": updateDir,
+                    "installErrorStr": installErrorStrArr,
+                    "bracketErrorStr": bracketErrorStrArr,
+                    "encoding": ""
```

Comment:
Please explain the reasoning behind the default encoding on both the platforms. 
On windows, there are two log files that we create, one is generated by wix, and the other is generated by me using the shell. Please verify the encodings used to create the files, and use them appropriately.
On mac, one is written by me using bash command, and the other using a native shell command. Verify the encoding here as well.

I say this because, if you are sure about the encodings then you can read them on the node side using

` fs.readFile(filename, { 'encoding' :` **_Insert appropriate encoding here_**` });`

Because right now you seem to be reading the data as a buffer, and that might not be required.